tests/lints/untyped-map-member.rs

```diff
@@ -0,0 +1,27 @@
+//! Tests for the `untyped-map-member` lint.
+
+use indoc::indoc;
+
+use pretty_assertions::assert_eq;
+
+use crate::util::lint_report;
+
+
+#[test]
+fn basic_sizeof() {
+    let code = indoc! { r#"
+    struct {
+        int a;
+        __uint(a, sizeof(b));
+    } name;
+    "# };
+
+    let expected = indoc! { r#"warning: [untyped-map-member] __uint(a, sizeof(b)) does not contain potentially relevant type information, consider using __type(a, b) instead
+  --> <stdin>:2:4
+  | 
+2 |     __uint(a, sizeof(b));
+  |     ^^^^^^
```

Comment:
I think we should hightlight the entire `__uint(a, sizeof(b));` construct. The message isn't really specific to the `__uint` part only. I suspect you will need https://github.com/d-e-s-o/bpflint/commit/37ec7ff9420050ef5bfea9501f4ecfbb5ec66416 to make that happen.